kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/LogReplay.java

```diff
@@ -64,15 +62,15 @@ public class LogReplay implements Logging {
      * Note that we don't need to read the entire RemoveFile, only the path and dv info.
      */
     public static final StructType ADD_REMOVE_READ_SCHEMA = new StructType()
-        .add("add", AddFile.READ_SCHEMA)
+        .add("add", AddFile.SCHEMA)
```

Comment:
we should add a todo: eventually, we won't want to read the entire AddFile schema. For example, if there is no filter on our query, then we don't need to read the AddFile stats

Reply:
added